common/include/Utilities/gtkGuiTools.h

```diff
@@ -89,45 +89,3 @@ static GtkWidget *ps_gtk_vseparator_new()
     return gtk_separator_new(GTK_ORIENTATION_VERTICAL);
 #endif
 }
-
-// These two routines have been rewritten over and over. May as well include a copy.
-// Renaming so as not to interfere with existing versions.
```

Comment:
This file was actually written by me within the last couple months. Anything not used in it is because I haven't used it yet.  In particular, as the comment says, these functions have been written over and over in the plugins. They just haven't been *replaced* in those locations yet.

Reply:
I'll exclude the changes from this file then.